src/utils/daysBetween.ts

```diff
@@ -0,0 +1,10 @@
+export const daysBetween = function (date1: number, date2: number) {
```

Comment:
Hmm, may be time to use `moment` ... 
```
const span = moment.duration(toDate.diff(fromDate)).asDays();
```

Reply:
Moment has been deprecated and is not advised for new projects, I could include something like date-fns but I think for now I'll hold off from adding a date library just for this.